README.md

```diff
@@ -258,6 +258,24 @@ const search = '?ft=foo,bar,baz';
 getBrowserQueryFeatures(search); // ['foo', 'bar', 'baz']
 ```
 
+#### getCurrentActiveFeatures
+
+Takes an array of initialFeatures, a req object, and a `window.location.search` string and returns an array of active Features. If search is not provided will grab the global `window.location.search` if available.
+
+`({ initialFeatures = [...String], req? , search? }) => [...String])]`
+
+```js
+const initialFeatures = ['cat', 'bar'];
+const req = {
+    query:{
+      ft='fiz,bat,help'
+    }
+  };
+const search = '?ft=foo,bar,baz';
+
+getCurrentActiveFeatures(initialFeatures, req, search); // ['cat', 'bar', 'fiz', 'bat', 'help', 'foo', 'baz']
```

Comment:
The argument should be an object
```
getCurrentActiveFeatures({ initialFeatures, req, search }); // ['cat', 'bar', 'fiz', 'bat', 'help', 'foo', 'baz']
```
Lets also show a call with only initial features as its the only required property.